src/main/kotlin/com/hibob/academy/dao/tables.kt

```diff
@@ -0,0 +1,28 @@
+package com.hibob.academy.dao
+
+import com.hibob.academy.utils.JooqTable
+
+class PetTable(tableName: String = "pets"): JooqTable(tableName){
+    val petId = createBigIntField("id")
+    val petName = createVarcharField("name")
+    val petType = createVarcharField("type")
+    val companyId = createBigIntField("company_id")
+    val dateOfArrival = createLocalDateField("date_of_arrival")
+
+    companion object{
+        val  instance = PetTable()
+    }
+}
+
+
+class OwnerTable(tableName: String = "owner"): JooqTable(tableName){
+    val ownerId = createBigIntField("owner_id")
+    val ownerName = createVarcharField("name")
+    val companyId = createBigIntField("company_id")
+    val employeeId = createBigIntField("employee_id")
```

Comment:
Im pretty sure that employeeId should be string field, correct me if im wrong.

Reply:
You right, my bad